eq-author-api/scripts/test.sh

```diff
@@ -33,6 +34,18 @@ if [ ! -f "data/QuestionnaireList.json" ]; then
     echo "[]" > "data/QuestionnaireList.json";
 fi
 
+if [ ! -f "data/UserList.json" ]; then 
```

Comment:
I think this can be removed as we are not supporting the file system data store any more.